int8 up_down_status

---
bool result